### ISSUE TITLE
Alternative way to log in.

### ISSUE THREAD

**Issue by stas-prze** [dosdude1/discord-lite#12]
Hello.
I'm a blind VoiceOver user and I'm unable to login due to the Captcha.
Would there be a possibility of adding an alternative login method, such as logging in directly with an account token salvaged from browser cookies, ETC?

**Comment by dosdude1** [comment]
Yes, I was planning on doing something like that. It needs to be done anyways for Tiger, where the captcha doesn't work at all.
